### ISSUE TITLE
Write a utility function to fetch arXiv IDs given a title

### ISSUE THREAD

**Issue by arunkannawadi** [arunkannawadi/independent-citation-counter#1]
Since arXiv identifier is not available readily in the Google Scholar page, we should query the arXiv database and fetch a publication's arXiv id from its title, and may be its first author.

**Comment by kabs9000** [comment]
@arunkannawadi Hi, did you ever make this?

**Comment by arunkannawadi** [comment]
No, I haven't had the chance to work on this repo but happy to accept PRs.
